backend/src/graphql/resolver/TransactionResolver.test.ts

```diff
@@ -67,6 +67,7 @@ describe('send coins', () => {
 
   describe('unknown recipient', () => {
     it('throws an error', async () => {
+      jest.clearAllMocks()
```

Comment:
no please don't use a general "before each" to clearAllMocks, because this will lead to empty the logger-messages before they could be checked in the afterwards test after the previous testcase.
So the clearing of logger messages pe clearAllMocks have to be done individually per testcase and only if necessary